lib/apple/index.js

```diff
@@ -26,7 +26,14 @@ function getReceiptFieldValue(receipt, field) {
 	if (receipt.hasOwnProperty('in_app') && receipt.in_app[0] && receipt.in_app[0].hasOwnProperty(field)) {
 		return receipt.in_app[0][field];
 	}
-	
+
+	return null;
+}
+
+function parseTime(time) {
```

Comment:
The implementation of this function seems incomplete.

`purchase_date_ms` sounds like a timestamp, but `expires_date` supposedly is an RFC 3339 date.

Reply:
Right, but `parseTime` converts integer-strings to integers...

So, I wasn't suggesting use this or that, but rather, the way you call this function now is with values that have a very different format (one is an integer-string, another is a formatted data/time-string).

Reply:
Both `expires_date` and `purchase_date_ms` are integer strings.

Apple docs are a bit confusing. See here:
https://developer.apple.com/library/content/releasenotes/General/ValidateAppStoreReceipt/Chapters/ReceiptFields.html

```
Subscription Expiration Date
The expiration date for the subscription, expressed as the number of milliseconds since January 1, 1970, 00:00:00 GMT.

ASN.1 Field Type 1708

ASN.1 Field Value IA5STRING, interpreted as an RFC 3339 date

JSON Field Name expires_date

JSON Field Value string, interpreted as an RFC 3339 date
```

The expiration date for the subscription, expressed as the ***number of milliseconds*** since January 1, 1970, 00:00:00 GMT.

See the receipt above again. Or am I missing something?

Reply:
Oh wow, indeed. My bad. Also Apple's bad it seems ;)

Reply:
Yeah its pretty strange... but as long as it works 👍 

Do you want me to change anything or just leave it how it is?

Reply:
I guess this is fine then :)